src/pages/Home/components/TeamBlock/TeamSlider/TeamSlider.tsx

```diff
@@ -0,0 +1,91 @@
+import { TeamPerson } from '@pages/Team/components/TeamPerson/TeamPerson';
+import React from 'react';
+import {
+  TABLET_POINT,
+  MOBILE_POINT,
+  MOBILE_SMALL_POINT,
+  TEAM_PERSONS
+} from '../../../../../constants';
+import { Swiper, SwiperRef, SwiperSlide } from 'swiper/react';
+import { Icon } from '@components/Icon/Icon';
+
+interface TeamSliderProps {}
+
+const TeamSlider: React.FC<TeamSliderProps> = () => {
+  const swiperRef = React.useRef<SwiperRef | any>(null); // delete the “any” type, how to solve the problem?
+  const [slidePerView, setSlidePerView] = React.useState<
+    1 | 2 | 3 | 4 | 'auto'
+  >(4);
+
+  React.useEffect(() => {
```

Comment:
use "breakpoints" props for the slider to change slides count instead of listening window resize event
For example this code will do the same as yours:
<Swiper
  spaceBetween={20} // should be 80, as in Figma layout
  grabCursor
  ref={swiperRef}
  keyboard={{ enabled: true }}
  slidesPerView={slidePerView}
  breakpoints={{
    [TABLET_POINT]: {
      slidesPerView: 4
    },
    [MOBILE_POINT]: {
      slidesPerView: 3
    },
    [MOBILE_SMALL_POINT]: {
      slidesPerView: 2
    },
    0: {
      slidesPerView: 1
    }
  }}
>